boards/arm/nrf9160_innblue22/doc/index.rst

```diff
@@ -0,0 +1,106 @@
+.. _nrf9160_innblue22:
+
+nRF9160-INNBLUE22
+#################
+
+Overview
+********
+
+The "innblue v2.2" is a cellular IoT sensor development board, which
```

Comment:
The page uses "INNBLUE22", "innblue v2.2", and "innblue22" as the board's name. It would be nice to pick one and be consistent. I see that nRF9160-DK is guilty of the same offense in its documentation, so this is a non-blocking request. I'll send a PR fixing the DK documentation next.

Reply:
> I'll send a PR fixing the DK documentation next.

https://github.com/zephyrproject-rtos/zephyr/pull/24002